xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -42,7 +42,6 @@
 #include "settings/Settings.h"
 #include "FileItem.h"
 #include "guilib/LocalizeStrings.h"
-#include "utils/StringUtils.h"
```

Comment:
Util.h includes StringUtils, thx for noticing, I removed it from VideoInfoScanner too.